resources/js/calendar-widget.js

```diff
@@ -196,6 +224,10 @@ export default function calendarWidget({
                 }
             };
 
+            settings.eventAllUpdated = (info) => {
```

Comment:
Remove this call, it's an experimental feature which could be changed at any time. There is no need to store the current view globally, since all the events receive the view object. If needed, this can be added individually on a per-project basis using getOptions already.

Reply:
Could you make an example on how to use such logic using getOptions? normally i would write some javascript but in this case it's php we write, right?

Reply:
I was wrong, since getOptions returns an array, this is currently not possible.